packages/world-module-erc20/src/Pausable.sol

```diff
@@ -0,0 +1,126 @@
+// SPDX-License-Identifier: MIT
+// Adapted from OpenZeppelin Contracts (utils/Pausable.sol)
+
+pragma solidity ^0.8.20;
+
+import { ResourceId, ResourceIdLib } from "@latticexyz/store/src/ResourceId.sol";
+
+import { Paused as PausedTbl } from "./codegen/tables/Paused.sol";
```

Comment:
nit: generally trying to avoid abbreviations whenever possible
```suggestion
import { Paused as PausedTable } from "./codegen/tables/Paused.sol";
```